src/Microsoft.OpenApi.OData.Reader/Microsoft.OpenAPI.OData.Reader.csproj

```diff
@@ -21,8 +21,13 @@
     <PackageTags>Microsoft OpenApi OData EDM</PackageTags>
     <RepositoryUrl>https://github.com/Microsoft/OpenAPI.NET.OData</RepositoryUrl>
     <PackageReleaseNotes>
-    - Further fix for generating unique operation ids for navigation property paths with composable overloaded functions #596
-    </PackageReleaseNotes>
+		- Upgraded to Microsoft.Odata.Edm 8.0.0
```

Comment:
those are changelogs from v2, I believe something went wrong in your cherry-pick, you should revert this and add an entry specific to your change + version bump